packages/cozy-harvest-lib/src/components/KonnectorAccounts.jsx

```diff
@@ -110,7 +109,7 @@ export class KonnectorAccounts extends React.Component {
     if (this.state.boundaryErrorInfo) {
       return (
         <div className="u-error">
-          <Bold>{this.state.boundaryError.message}</Bold>
+          <b>{this.state.boundaryError.message}</b>
```

Comment:
Why not <Typography variant="h6" > as stated here in the ui's doc: 
<img width="733" alt="Capture d’écran 2021-01-13 à 09 06 44" src="https://user-images.githubusercontent.com/1107936/104423717-bc666700-557e-11eb-9e80-7420f229cab5.png">

Reply:
Done